core/spoofax.core/src/main/java/mb/spoofax/core/language/testrunner/MultiTestSuiteRun.java

```diff
@@ -0,0 +1,92 @@
+package mb.spoofax.core.language.testrunner;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Container for multiple {@link TestSuiteRun}s.
+ */
+public class MultiTestSuiteRun implements Serializable {
+
+    public final List<TestSuiteRun> suites = new ArrayList<>();
+
+    private int numFailed = 0;
```

Comment:
Because this class can be used as an output in PIE, it must be immutable. These should be final and it should not be possible to add `TestSuiteRun`s after construction. Consider using the `org.immutables` annotation processor to create an immutable data type with a builder. See `CommandRequest` for an example.